tests/integration/backward_compatible/authentication_tests/authentication_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from hazelcast.errors import HazelcastError
+from tests.base import HazelcastTestCase
+from tests.util import get_abs_path, set_attr
+from hazelcast.client import HazelcastClient
+from hazelcast.token_provider import BasicTokenProvider
```

Comment:
For the same reason (backward compatibility tests), we should put this import in a try-except block

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6